src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -606,7 +604,7 @@ const ContextMenuActions: ContextMenuAction[] = [
         successIcon: Expensicons.Checkmark,
         shouldShow: ({type, isProduction}) => type === CONST.CONTEXT_MENU_TYPES.REPORT && !isProduction,
         onPress: (closePopover, {reportID}) => {
-            const report = ReportConnection.getAllReports()?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
+            const report = ReportUtils.getReport(reportID);
```

Comment:
Left `ReportUtils.getReport` in this file, cause it's already used there, and changing it gives the error:
```
Only call Onyx.connect() from inside a /src/libs/** file. React components and non-library code should not use Onyx.connect()  rulesdir/prefer-onyx-connect-in-libs]
```